src/inbound.rs

```diff
@@ -0,0 +1,34 @@
+use hyper::{Body, Client, Uri};
+use serde_derive::{Deserialize, Serialize};
+
+// TODO return result here
+async fn body_to_string(body: Body) -> String {
+    let body_bytes = hyper::body::to_bytes(body).await.unwrap();
+    String::from_utf8(body_bytes.to_vec()).unwrap()
+}
+
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct Quote {
+    price: u32,
+    size: u32,
+    duration: u32,
+    address: String,
+}
+
+pub async fn get_quote(pubkey: &str, refund_address: &str) -> Result<Quote, hyper::http::Error> {
+    let https = hyper_tls::HttpsConnector::new();
+    let client = Client::builder().build::<_, hyper::Body>(https);
+    // get address
+    // suggest capacity
+    let base_uri = format!(
+        "https://nolooking.chaincase.app/api/getinbound?nodeid={}&capacity={}&duration={}&refund_address={}",
```

Comment:
Awesome that refund is trivial to add since it comes from nolooking user's node node 😍

Reply:
It's almost spooky ahahaha
that's why i added "Send refund address" in the UI so it doesn't dodgy